3_navigation.ipynb

```diff
@@ -232,26 +481,42 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>課題</b>\n",
     "    <p>ある場所についたら次のゴールに自動的に向かうようにして、部屋を周回するプログラムを書いてみましょう。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Write a code to move to the position to the another to move around the room.</p>\n",
```

Comment:
What about something like this:

Let's try to write a program that goes around a room, then automatically goes to its next goal after arriving at a certain place.